### PR TITLE
Feat/upgrade to version5

### PR DESCRIPTION
**Pull request by dhruvTor**
Spacing and indentation corrected, branch name changed,Header comments removed from all files.

### CODE REVIEW

File: Sources/FetchNodeDetails/Contract/abi.swift [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Foundation
+
+var contractABIString = "[{\"inputs\":[{\"internalType\":\"string\",\"name\":\"_verifier\",\"type\":\"string\"},{\"internalType\":\"bytes32\",\"name\":\"hashedVerifierId\",\"type\":\"bytes32\"}],\"name\":\"getNodeSet\",\"outputs\":[{\"internalType\":\"uint256\",\"name\":\"currentEpoch\",\"type\":\"uint256\"},{\"internalType\":\"string[]\",\"name\":\"torusNodeEndpoints\",\"type\":\"string[]\"},{\"internalType\":\"uint256[]\",\"name\":\"torusNodePubX\",\"type\":\"uint256[]\"},{\"internalType\":\"uint256[]\",\"name\":\"torusNodePubY\",\"type\":\"uint256[]\"},{\"internalType\":\"uint256[]\",\"name\":\"torusIndexes\",\"type\":\"uint256[]\"}],\"stateMutability\":\"view\",\"type\":\"function\"}]"
```
Comment (chaitanyapotti) [inline Sources/FetchNodeDetails/Contract/abi.swift:3]: Prefer importing json and converting it to string on demand

File: Sources/FetchNodeDetails/Contract/TorusLookup.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+public enum TorusLookupContract{
+    public struct getNodeSet:ABIFunction{
+        public static var name: String = "getNodeSet"
+        public var gasPrice: BigUInt?
+        public var gasLimit: BigUInt?
+        public var contract: EthereumAddress
+        public var from: EthereumAddress?
+        public var verifier:String
+        public var hashedVerifierId:Data
```
Comment (chaitanyapotti) [inline Sources/FetchNodeDetails/Contract/TorusLookup.swift:14]: Add linting everywhere and follow consistent pattern

File: README.md [hunk 3]
```diff
@@ -14,7 +14,9 @@ A description of this package.
 
 ```swift
 
-EthereumNetwork.Mainnet = 0x638646503746d5456209e33a2ff5e3226d698bea // torus mainnet 
-EthereumNetwork.Ropsten = 0x4023d2a0D330bF11426B12C6144Cfb96B7fa6183 // torus testnet
+EthereumNetwork.MAINNET = 0xf20336e16B5182637f09821c27BDe29b0AFcfe80 // torus mainnet 
+EthereumNetwork.ROPSTEN = 0x4023d2a0D330bF11426B12C6144Cfb96B7fa6183 // torus polygon
+EthereumNetwork.POLYGON = 0x6258c9d6c12ed3edda59a1a6527e469517744aa7 // torus ropsten
```
Comment (chaitanyapotti) [inline README.md:19]: This must be POLYGON_MAINNET i think. Please refer to web sdk and change it

File: Sources/FetchNodeDetails/FetchNodeDetails+promise.swift [hunk 4]
```diff
@@ -0,0 +1,85 @@
+import Foundation
+import web3
+import BigInt
+import PromiseKit
+import OSLog
+@testable import web3
+
+extension FetchNodeDetails {
+    public func getAllNodes(skip:Bool = false,verifier:String,verifierID:String) -> Promise<AllNodeDetailsModel>{
```
Comment (chaitanyapotti) [inline Sources/FetchNodeDetails/FetchNodeDetails+promise.swift:9]: Rename it to match web and android

File: Sources/FetchNodeDetails/FetchNodeDetails.swift [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    var proxyAddress : EthereumAddress
+    var projectID:String = "7f287687b3d049e2bea7b64869ee30a3"
+    let yourContractABI: String = contractABIString
+    var urlSession : URLSession
+    var updated = false
+    var nodeDetails:AllNodeDetailsModel{
+        return .init(_currentEpoch: self.currentEpoch, _nodeListAddress: self.proxyAddress.value, _torusNodeEndpoints: self.torusNodeEndpoints, _torusIndexes: self.torusIndexes, _torusNodePub: self.torusNodePub, _updated:self.updated)
+    }
+    
+    public init(proxyAddress: String, network: EthereumNetwork, logLevel: OSLogType = .default, urlSession : URLSession = URLSession.shared){
```
Comment (chaitanyapotti) [inline Sources/FetchNodeDetails/FetchNodeDetails.swift:63]: these parameters are optional. Pls refer to web sdk

Comment (chaitanyapotti) [reply Sources/FetchNodeDetails/FetchNodeDetails.swift:63]: Also, add support for sending in a string

**Review by chaitanyapotti — CHANGES_REQUESTED** [review]
pls work on the changes

File: Sources/FetchNodeDetails/FetchNodeDetails+promise.swift (hunk 4 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/FetchNodeDetails+promise.swift:9]: Done

File: Sources/FetchNodeDetails/FetchNodeDetails.swift (hunk 5 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/FetchNodeDetails.swift]: Done

File: README.md (hunk 3 above)
Comment (dhruvTor) [reply README.md:19]: It's  POLYGON

File: Sources/FetchNodeDetails/Contract/TorusLookup.swift (hunk 2 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/Contract/TorusLookup.swift:14]: Done

File: Sources/FetchNodeDetails/Contract/abi.swift (hunk 1 above)
Comment (dhruvTor) [reply Sources/FetchNodeDetails/Contract/abi.swift]: Done

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**